src/js/modules/PageBreadcrumbs.js

```diff
@@ -5,13 +5,13 @@ const PageBreadcrumbs = ({ title, order, ...rest }) => {
 
   const breadcrumbs = order.map(breadcrumb => ({
     className: breadcrumb,
-    title: rest[breadcrumb].title,
-    slug: rest[breadcrumb].slug,
+    text: rest[breadcrumb].text,
+    slug: `/${breadcrumb}s/${rest[breadcrumb].id}`,
```

Comment:
Oh wait, can we use slug here? Is this for theme or topic?

Reply:
it's the same for both topic and theme since I'm just loo[ing through all links in the breadcrumb list.

Reply:
Why move to IDs? I thought we were moving towards slugs for all.

Reply:
Not moving to Id's -- I'm preserving the ids we had before so navigation still works. Our queries currently fetch topic page data via id. I think that's how the Joplin queries are set up for topic data.